### PR TITLE
json-parser: recognize SARIF format produced by gitleaks

### PR DESCRIPTION
**Pull request by kdudka**

### CODE REVIEW

File: tests/csgrep/CMakeLists.txt [hunk 1]
```diff
@@ -143,3 +143,4 @@ test_csgrep("96-sarif-levels"                         )
 test_csgrep("97-sarif-set-tool"                       )
 test_csgrep("98-json-parser-shellcheck"               )
 test_csgrep("99-csparser-his-metrics"                 )
+test_csgrep("0100-sarif-parser-gitleaks"              )
```
Comment (lzaoral) [inline tests/csgrep/CMakeLists.txt:146]: This is not directly related to this PR, but wouldn't it be better to add leading zeros also to all existing tests?

**Review by lzaoral — APPROVED** [review]
LGTM!

File: tests/csgrep/CMakeLists.txt (hunk 1 above)
Comment (kdudka) [reply tests/csgrep/CMakeLists.txt:146]: Absolutely.  I will open a separate pull request for the rename as soon as there are no other open pull requests, to avoid unnecessary conflicts.

**Comment by kdudka** [comment]
@lzaoral Thanks for review!
